extensions/amp-list/0.1/amp-list.js

```diff
@@ -378,6 +402,31 @@ export class AmpList extends AMP.BaseElement {
 }
 
 
+/**` @private */
+function handleFallbackSuccess_() {
+  if (this.getFallback()) {
+    // Hide in case fallback was displayed for a previous fetch.
+    this.toggleFallbackInMutate_(false);
+  }
+  this.togglePlaceholder(false);
+  this.toggleLoading(false);
+}
+
+/**
+ * @param {!Error} error
+ * @private
```

Comment:
Add `@throws` with description.

Reply:
I don't think we actually use that throughout the codebase.